### ISSUE TITLE
Add 'all_accounts' parameter to wallet rpc 'get_address' and 'incoming_transfers'

### ISSUE THREAD

**Issue by woodser** [monero-project/monero#5362]
This issue requests the addition of the `all_accounts` parameter in wallet rpc `get_address` and `incoming_transfers` similar to `get_balance` and `get_transfers` (pr #5150).  Otherwise, a client must make one request per account to each of these endpoints in order to collect information across all accounts.  This becomes expensive as the number of accounts grows and especially if the requests are non-local.

Related to #5109.

**Comment by jtgrassie** [comment]
You mean like PR's #5154 and #5156 right? Also, is there a need for changing `incoming_transfers` given that `get_transfers` already get's incoming transfers too?

**Comment by woodser** [comment]
Right.  As I understand it, `incoming_transfers` returns the key image, spent status, and index of incoming outputs.  These fields are not returned from `get_transfers`.
